packages/mobile/src/screens/collection-screen/CollectionScreen.tsx

```diff
@@ -48,6 +50,7 @@ import { makeStyles } from 'app/styles'
 import { CollectionScreenDetailsTile } from './CollectionScreenDetailsTile'
 import { CollectionScreenSkeleton } from './CollectionScreenSkeleton'
 
+const { open: openEditPlaylist } = createPlaylistModalUIActions
```

Comment:
so how come this is edit, should be create?

Reply:
I think the same actions are used for creating and editing. this is for the edit button so called it openEditPlaylist to be clear